apps/institution-service/openApi.json

```diff
@@ -761,6 +761,118 @@
             }
           }
         }
+      },
+      "post": {
```

Comment:
It looks like this is attached to the wrong url. When I load it in the swagger editor it says it requires an id

<img width="623" alt="image" src="https://github.com/user-attachments/assets/81c4d6b9-316e-4384-9939-66a3c63e121a">